### PR TITLE
feat: support deploy via docker compose

### PR DESCRIPTION
**Pull request by manast**

### CODE REVIEW

File: packages/widged/src/widget-view.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const WidgetView = () => {
         height: '100%',
       },
       config: {
-        id: 'my-config-id',
+        id: 1234,
```
Comment (arokor) [inline packages/widged/src/widget-view.tsx:42]: A constant maybe?

**Review by arokor — APPROVED** [review]
Looks good! Maybe add a section about how to run this in README.md

File: packages/widged/src/widget-view.tsx (hunk 1 above)
Comment (manast) [reply packages/widged/src/widget-view.tsx:42]: yes. I think the id should be optional.

**Comment by manast** [comment]
> Looks good! Maybe add a section about how to run this in README.md

Yes, it is coming in a separate commit, almost ready.

**Pull request merged**